### ISSUE TITLE
Release new LO, ARM, 7.3

### ISSUE THREAD

**Issue by jpike88** [shelfio/libreoffice-lambda-layer#46]
I'm struggling to compile myself a new version of libreoffice 7.3 for Lambda running on ARM Architecture.

@vladgolubev as you have the most experience working with this stuff, are you able to maybe spend a few hours and help us all out? Instead of trying to tackle every weird edge error people are bringing up, we can all start fresh with a new version, complete with all the fixes and performance improvements etc, I'm sure that would put a lot of the drama people are having to bed.

If you don't have the time to take it all on yourself, are you able to help test/guide things through to a point?

**Comment by dallanmc** [comment]
I wonder would it be possible to just stick full libreoffice on an EFS which could be accessed by the lambda?

**Comment by vladgolubev** [comment]
@dallanmc possible but you'll encounter tremedeous FS latencies due to the nature of NFS and the number of files read by Libreoffice. It has like thousands of files, and each NFS read and write operation adds millisecond-level latency

**Comment by vladgolubev** [comment]
@jpike88 I compiled 7.3 for x86_64 as a Lambda Docker Image

https://github.com/shelfio/libreoffice-lambda-base-image

**Comment by dallanmc** [comment]
> @dallanmc possible but you'll encounter tremedeous FS latencies due to the nature of NFS and the number of files read by Libreoffice. It has like thousands of files, and each NFS read and write operation adds millisecond-level latency

Reading around that seemed to be the major drawback. Thanks for the reply.

**Comment by dallanmc** [comment]
> @jpike88 I compiled 7.3 for x86_64 as a Lambda Docker Image
> 
> https://github.com/shelfio/libreoffice-lambda-base-image

This looks great  Would starting this docker image be quicker than unpacking the layer on a cold start? And what about subsequent invocations on a warm start. Would it be quicker or slower than executing from /tmp

**Comment by vladgolubev** [comment]
@dallanmc yes, starting a docker image is generally faster than unpacking a layer

and it will be even faster after this fix https://github.com/shelfio/libreoffice-lambda-base-image/issues/4

**Comment by dallanmc** [comment]
> @dallanmc yes, starting a docker image is generally faster than unpacking a layer
> 
> and it will be even faster after this fix [shelfio/libreoffice-lambda-base-image#4](https://github.com/shelfio/libreoffice-lambda-base-image/issues/4)

Looks great! Thanks for the reply.
